app/src/main/res/layout/subscription_item.xml

```diff
@@ -6,42 +6,39 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
+    android:padding="4dp"
+    android:elevation="4dp"
     android:foreground="?attr/selectableItemBackground">
 
     <de.danoeh.antennapod.ui.common.SquareImageView
         android:id="@+id/imgvCover"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:background="@color/non_square_icon_background"
         android:scaleType="fitCenter"
+        android:outlineProvider="background"
         squareImageView:direction="width"
         tools:src="@tools:sample/avatars" />
 
-    <com.joanzapata.iconify.widget.IconTextView
+    <TextView
```

Comment:
My idea in #6094 was to duplicate the TextViews. One for the fallback image and one for the normal title. Then we no longer need to move around the TextView in Java:

https://github.com/AntennaPod/AntennaPod/blob/fbfd7c43ac1d6970acf43c2ef2a7819a9150e15e/app/src/main/java/de/danoeh/antennapod/adapter/SubscriptionsRecyclerAdapter.java#L75-L92